R/rmarkdown.R

```diff
@@ -50,3 +53,15 @@ render_rmarkdown <- function(pkg, input, output, ..., copy_images = TRUE, quiet
 
   invisible(path)
 }
+
+#' Get BSTINPUTS for JSS articles (adapted from tools::texi2dvi)
```

Comment:
* Please don't use roxygen comments here. 
* And I think the purpose is obvious from the name, so just mention the source